src/main/resources/static/js/login.js

```diff
@@ -0,0 +1,20 @@
+let formElem = document.querySelector('#form');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/koreait0000/MNDC/pullRequest?prid=7786920)
```suggestion
let formElem = document.querySelector("#form");
```